AeroGearOAuth2/OAuth2Module.swift

```diff
@@ -133,7 +134,12 @@ open class OAuth2Module: AuthzModule {
                 self.webView!.targetURL = url
                 config.webViewHandler(self.webView!, completionHandler)
             } else {
-                UIApplication.shared.openURL(url)
+                if #available(iOS 9.0, *) {
```

Comment:
@corinnekrych Another option is to turn `Config.isWebView` into an enum rather than boolean. Then we could check if it's a web view, Safari web view, or external browser.